inspirehep/modules/records/api.py

```diff
@@ -30,15 +30,45 @@
 from elasticsearch.exceptions import NotFoundError
 
 from invenio_pidstore.errors import PIDDoesNotExistError
+from invenio_pidstore.models import PersistentIdentifier
 from invenio_records.api import Record
+from invenio_db import db
 
 from inspirehep.utils.record_getter import (
     RecordGetterError,
     get_es_record_by_uuid
 )
 
 
-class ESRecord(Record):
+class InspireRecord(Record):
+
+    """Record class that fetches records from DataBase."""
+
+    def merge(self, other):
+        """Merge all pidstores of deleted record to the merged record."""
```

Comment:
To better document the `other` parameter, you can use this syntax https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/search/api.py#L50-L53